packages/@react-native-windows/tester/js/examples-win/DisplayNone/DisplayNoneExample.windows.tsx

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
this file shouldn't be checked in - should only include files under the /src directory. This one is the result of running yarn build which copies files to where metro pickes it up from

Reply:
good catch; the tsx extension fooled me. Is this file needed in the compiled output or only the .js?